`NoClassDefFoundError` when running binaries generated by Compose Multiplatform Gradle plugin

I get the error below 

```
Exception in thread "main" java.lang.NoClassDefFoundError: java/lang/management/ManagementFactory
        at com.github.ajalt.mordant.internal.MppInternal_jvmKt.runningInIdeaJavaAgent(MppInternal.jvm.kt:57)
        at com.github.ajalt.mordant.terminal.TerminalDetection.isIntellijRunActionConsole(TerminalDetection.kt:199)
        at com.github.ajalt.mordant.terminal.TerminalDetection.detectTerminal(TerminalDetection.kt:19)
        at com.github.ajalt.mordant.terminal.StdoutTerminalInterface.<init>(StdoutTerminalInterface.kt:17)
        at com.github.ajalt.mordant.terminal.Terminal.<init>(Terminal.kt:52)
        at com.github.ajalt.mordant.terminal.Terminal.<init>(Terminal.kt:41)
        at org.example.MainKt.main(Main.kt:7)
Caused by: java.lang.ClassNotFoundException: java.lang.management.ManagementFactory
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(Unknown Source)
        at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(Unknown Source)
        at java.base/java.lang.ClassLoader.loadClass(Unknown Source)
        ... 7 more
```

when using the following Main function

```kotlin
import com.github.ajalt.mordant.terminal.Terminal

fun main() = Terminal().println("test")
```

and the following build.gradle.kts file

```kotlin
import org.jetbrains.compose.desktop.application.dsl.TargetFormat

plugins {
    kotlin("jvm") version "2.0.0"
    id("org.jetbrains.compose") version "1.6.11"
    id("org.jetbrains.kotlin.plugin.compose") version "2.0.20"
}

group = "org.example"
version = "1.0.0"

repositories {
    mavenCentral()
    google()
}

dependencies {
    implementation("com.github.ajalt.mordant:mordant:2.7.2")
    testImplementation(kotlin("test"))
    implementation("androidx.compose.runtime:runtime:1.6.8")
}

tasks.test {
    useJUnitPlatform()
}

kotlin {
    jvmToolchain(17)
    sourceSets {
        val main by getting
        val test by getting
    }
}

compose.desktop {
    application {
        mainClass = "org.example.MainKt"

        nativeDistributions {
            targetFormats(TargetFormat.Pkg, TargetFormat.Exe, TargetFormat.Deb)
        }
    }
}
```

The full project is attached. I have tested this on both my Windows 11 computer and a WSL instance running Ubuntu 22.04.1 LTS.
[mordant_bug.zip](https://github.com/user-attachments/files/16749963/mordant_bug.zip)

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -22,6 +22,7 @@
 ### Fixed
 - Fixed ConcurrentModificationException from progress bars when updated under very high concurrency [(#204)](https://github.com/ajalt/mordant/issues/204)
 - Improved performance of progress bars under high concurrency. [(#207)](https://github.com/ajalt/mordant/issues/207)
+- Fixed `NoClassDefFoundError` when running with certain gradle plugins [(#217)](https://github.com/ajalt/mordant/issues/217)
 
 ## 2.7.2
 ### Fixed

--- a/mordant/src/jvmMain/kotlin/com/github/ajalt/mordant/internal/MppInternal.jvm.kt
+++ b/mordant/src/jvmMain/kotlin/com/github/ajalt/mordant/internal/MppInternal.jvm.kt
@@ -53,7 +53,7 @@ internal actual fun runningInIdeaJavaAgent() = try {
     val bean = ManagementFactory.getRuntimeMXBean()
     val jvmArgs = bean.inputArguments
     jvmArgs.any { it.startsWith("-javaagent") && "idea_rt.jar" in it }
-} catch (e: Exception) {
+} catch (e: Throwable) {
     false
 }
 

